weave/wandb_interface/wandb_stream_table.py

```diff
@@ -97,22 +21,25 @@ class _StreamTableSync:
     _project_name: str
     _entity_name: str
 
-    _artifact: WandbLiveRunFiles
+    _artifact: typing.Optional["Artifact"]
 
-    _weave_stream_table: StreamTableType
-    _weave_stream_table_ref: artifact_base.ArtifactRef
+    _weave_stream_table: typing.Any
+    _weave_stream_table_ref: typing.Any
 
     _client_id: str
 
     def __init__(
         self,
         table_name: str,
         *,
+        config: typing.Optional[dict] = None,
         project_name: typing.Optional[str] = None,
         entity_name: typing.Optional[str] = None,
-        _disable_async_file_stream: bool = False,
+        hidden: bool = True,
```

Comment:
Added a new hidden variable so users can use this within a regular workspace.  Need to test / think about this more.